modules/@apostrophecms/admin-bar/index.js

```diff
@@ -383,6 +385,33 @@ module.exports = {
           }
           return b.last === true ? -1 : 1;
         });
+      },
+
+      // Add custom context labels and place the ones
+      // that have `last: true` at the end
+      // of the list so that they will be
+      // displayed after the others.
+      //
+      // Example:
+      //
+      // ```js
+      // self.addContextLabel({
+      //   is: 'myLabel'
+      //   label: 'apos:myLabel',
+      //   tooltip: 'apos:myTooltip',
+      //   last: true,
+      //   modifiers: ['apos-is-warning', 'apos-is-filled']
+      // });
+      // ```
+      addContextLabel(label) {
```

Comment:
Any reason not to use this rather than directly manipulating the array in doc template library? It's a useful technique if you have to change something or there's a timing problem otherwise.

Reply:
Yes, because when using it, it adds new elements `getBrowserData` time in the array stored in backend.
In our case we just want to add it for specific pages (aka templates ones).
I'm even not sure this method makes sense for these laels (except if you want to add some for all pages). 
But we wanted to stick to what has been done for bars.